packages/wow-ui/src/components/Spinner/BlueSpinner.tsx

```diff
@@ -0,0 +1,40 @@
+import Lottie from "lottie-react";
+import type { CSSProperties } from "react";
+
+import blueSpinner from "@/assets/lottie/blueSpinner.json";
+
+/**
+ * @description 블루 스피너 컴포넌트입니다.
+ *
+ * @param {CSSProperties["width"]} [width] - 스피너의 너비.
+ * @param {CSSProperties["height"]} [height] - 스피너의 높이.
+ * @param {boolean} [loop] - 애니메이션의 반복 재생 여부. 기본값은 `true`.
+ * @param {boolean} [autoPlay] - 컴포넌트가 마운트될 때 자동으로 애니메이션이 재생될지 여부. 기본값은 `true`.
+ */
+export interface BlueSpinnerProps {
+  width?: CSSProperties["width"];
+  height?: CSSProperties["height"];
+  autoPlay?: boolean;
+  loop?: boolean;
+}
```

Comment:
p4;
lottie-react 에서 [Lottie type](https://github.com/Gamote/lottie-react/blob/main/src/types.ts) 도 제공해주고 있는데 그 타입을 확장해서 써보는 건 어떨까요?! `onComplete` 와 같이 애니메이션이 끝난 뒤에 실행시킬 함수도 넘겨볼 수 있을 거 같아용
```suggestion
import { Lottie,  LottieComponentProps } from 'lottie-react'; 

export interface BlueSpinnerProps export LottieComponentProps {
  width?: CSSProperties["width"];
  height?: CSSProperties["height"];
}
```

Reply:
굿굿 수정 완료입니다